joinfaces-starters/weld-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.joinfaces</groupId>
+        <artifactId>joinfaces-starters</artifactId>
+        <version>3.0.0.RC2-SNAPSHOT</version>
+    </parent>
+    <artifactId>weld-spring-boot-starter</artifactId>
```

Comment:
Aren't `weld-spring-boot-starter` and `omnifaces-spring-boot-starter` mutually dependent now?

Reply:
Hi Lars,
as per my understanding, joinfaces declares to support omnifaces 1.x. With this omnifaces version, Weld is not needed.

Futhermore, weld can be used for everything that needs CDI, and Omnifaces is not the only library that needs CDI. Potentially you can decide to use weld in order to make working libraries different than omnifaces

Reply:
Then we should have 2 omifaces starters, one for omnifaces 1.x and another one for the current omnifaces version.

Reply:
Are we sure? I'm asking for some reasons.

1. Currently switch from omnifaces 1 to omnifaces 3 is very easy. We only need to override the omnifaces.version property and add the dependency to the weld-spring-boot-starter.

2. Directly from the omnifaces site "For users on yet more outdated environments who can’t/won’t use CDI, use 1.14.1 instead. It doesn’t contain anything from CDI nor new things which were added in 2.x, but it does contain enhancements and bugfixes to existing 1.x things. Note that there is no 1.15 nor 1.16. The 1.14.1 is latest version of the 1.x branch which is also in maintenance mode." Seems that Omnifaces 1.x is abandoned. So if joinfaces decides to abandon Omnifaces as well, the second starter is not needed. 

So basically, I think I'm able to do that. the second starter. I'm only asking in order to be sure that is the direction that Joinfaces needs.